xbmc/video/windows/GUIWindowVideoNav.cpp

```diff
@@ -642,6 +642,15 @@ void CGUIWindowVideoNav::LoadVideoInfo(CFileItemList &items, CVideoDatabase &dat
       if (!pItem->m_bIsFolder && !fetchedPlayCounts)
       {
         database.GetPlayCounts(items.GetPath(), items);
+        // for addons if playcount is still -1 set it to zero
+        if(items.IsPlugin())
+        {
+          for (auto pItem : items)
```

Comment:
`pItem` is already used as a variable in this scope so you're accidentally shadowing it which we should avoid to not cause any confusion.